cloud_utils/cache/stores/redis_sync.py

```diff
@@ -35,21 +36,24 @@ def get_item(key: str):
             logging.debug(f"{key} is not in {name}")
             raise KeyError
         try:
+            if type(result) is bytes:
+                return result
             return json.loads(result)
         except ValueError:  # Key contents are malformed (will force key to update).
             logging.error(f"Malformed key detected: {key} in {name}.")
             raise KeyError
 
     def set_item(key: str, value):
+        value = json.loads(value) if type(value) is not bytes else value
```

Comment:
this should be json.dumps no?

as this is the return value in line 47

unless i'm missing something

Reply:
yes, should totally be json.dumps! got the two confused